src/api/AppActivity.js

```diff
@@ -0,0 +1,141 @@
+/**
+ * @flow
+ * @file Helper for the box App Activity API
+ * @author Box
+ */
+import MarkerBasedAPI from './MarkerBasedAPI';
+import {
+    ERROR_CODE_DELETE_APP_ACTIVITY,
+    FIELD_ACTIVITY_TEMPLATE,
+    FIELD_APP,
+    FIELD_CREATED_BY,
+    FIELD_OCCURRED_AT,
+    FIELD_RENDERED_TEXT,
+} from '../constants';
+
+const APP_ACTIVITY_FIELDS: string = [
```

Comment:
toString will do this for you as well. Also, we have a fields util method you could add this array to.